openmls/src/group/mls_group/mod.rs

```diff
@@ -407,8 +407,7 @@ impl MlsGroup {
     }
 
     /// Clear the pending proposals.
-    #[cfg(test)]
-    pub(crate) fn clear_pending_proposals(&mut self) {
+    pub fn clear_pending_proposals(&mut self) {
```

Comment:
```suggestion
    ///
    /// Warning: Once the pending proposals are cleared it will be impossible to process
    /// a Commit message that references proposals those proposals. Only use this
    /// function as a last resort, e.g. when a call to
    /// `MlsGroup::commit_to_pending_proposals` fails.
    pub fn clear_pending_proposals(&mut self) {
```

Reply:
7cc8b08